elife/docker-scripts/docker-prune

```diff
@@ -1,10 +1,15 @@
 #!/bin/bash
 set -e
 
-docker container prune
-docker volume prune
+docker container prune -f
+docker network prune -f
+docker volume prune -f
 
-#docker images prune
 echo "Clean up all images not used in the last 14 days"
+limit=$(date +%Y-%m-%d -d "14 days ago")
```

Comment:
Unrelated, but macOS uses BSD's `date` so has to be changed to `date -j -v -14d "+%Y-%m-%d"` to reuse.

Reply:
Is there a portable option to generate `2018-07-27`? This is usually deployed on Ubuntu Linux, other flavors may be used e.g. in EKS nodes that thave their own AMI.

Reply:
Could try https://stackoverflow.com/a/8748193/77199.

Reply:
Seems `docker image prune` supports `"until=240h"` so no `date` needed